TheJumpCloud/support
# SA-3064 — remove /Applications/PasswordManager if exists

## Issues
* [SA-3064](https://jumpcloud.atlassian.net/browse/SA-3064) - remove /Applications/PasswordManager if exists

## What does this solve?

Fast followup from yesterdays' changes to the password manager script. This change will remove the JumpCloud Password Manger from /Applications IF it was installed in a user directory /User/Username/Applications

## Is there anything particularly tricky?

## How should this be tested?

Run on a system with the JumpCloud Password Manager installed in the `/Applications/` directory, this script should re-install the Password Manager App in `/Users/Username/Applications` and remove the app from `/Applications/` if it exists.

## Screenshots
